profiler/src/ProfilerEngine/Datadog.Profiler.Native/StackSamplerLoopManager.cpp

```diff
@@ -231,7 +231,7 @@ void StackSamplerLoopManager::WatcherLoopIteration()
     if (_deadlockInterventionInProgress >= 1)
     {
         _deadlockInterventionInProgress++;
-        Log::Error("StackSamplerLoopManager::WatcherLoopIteration - Deadlock intervention still in progress for thread ", _pTargetThread->GetOsThreadId(),
+        Log::Info("StackSamplerLoopManager::WatcherLoopIteration - Deadlock intervention still in progress for thread ", _pTargetThread->GetOsThreadId(),
```

Comment:
maybe warning ?

Reply:
Depends how likely it is to happen, and what we/customer should do about it IMO. If the customer can't do anything (other than get worried), and we would only need to when investigating an escalation, then Info or even Debug makes sense to me?

Reply:
I would prefer Info : no need to see this log as a potential problem. It will probably happen for overloaded small containers as expected

Reply:
Yep, just discussed with Christophe, and I had a similar reasoning: I do not want Customer/support to be scared because there is a warning with "deadlock" in it.
I would like it to be there so if something happens, it will be a lead.
So we'll keep it info I think